packages/camera/camera_avfoundation/ios/camera_avfoundation/Package.swift

```diff
@@ -12,20 +12,29 @@ let package = Package(
     .iOS("12.0")
   ],
   products: [
-    .library(name: "camera-avfoundation", targets: ["camera_avfoundation"])
+    .library(
+      name: "camera-avfoundation", targets: ["camera_avfoundation", "camera_avfoundation_objc"])
   ],
   dependencies: [],
   targets: [
     .target(
       name: "camera_avfoundation",
+      dependencies: ["camera_avfoundation_objc"],
+      path: "Sources/camera_avfoundation",
+      resources: [
+        .process("Resources")
+      ]
+    ),
+    .target(
+      name: "camera_avfoundation_objc",
       dependencies: [],
-      exclude: ["include/camera_avfoundation-umbrella.h", "include/CameraPlugin.modulemap"],
```

Comment:
same question to the umbrella header.

Reply:
Correct me if I'm missing something but I don't see much value in having a custom umbrella header without a custom module map. Without the module map cocoapods generate the public umbrella header that includes everything either way